widgets/src/main/resources/org/pentaho/gwt/widgets/themes/public/themes/ruby/globalRuby.css

```diff
@@ -8258,6 +8258,47 @@ body.bootstrap .modal.confirmationDialog .dialog div.body {
   color: #7C0B2B;
 }
 
-.xul-label-disabled{
-	color: #666 !important;
+.xul-label-disabled {
+  color: #666 !important;
+}
+
+.gwt-ListBox:focus,.gwt-ListBox:hover {
+border: 1px solid #414141;
+}
+
+.pentaho-filechooseupbutton:focus,
+.pentaho-filechooseupbutton:hover{
+  outline: 1px solid #CC0000;
+}
+
+.fileChooserCell:focus{
+  outline: 1px solid #CC0000;
+}
+
+.fileChooser-scrollPanel:focus{
+  outline:1px solid #CC0000;
+}
+
+.custom-dropdown:focus,
+.custom-dropdown:hover{
+  background-color: #CC0000;
+  color: #FFFFFF;
+}
```

Comment:
Please refer to related discussions about focus visible.
- https://github.com/pentaho/pentaho-commons-gwt-modules/pull/819#discussion_r1071262057
- https://github.com/pentaho/pentaho-commons-gwt-modules/pull/819#discussion_r1071264610
- https://github.com/pentaho/pentaho-commons-gwt-modules/pull/819#discussion_r1071275564